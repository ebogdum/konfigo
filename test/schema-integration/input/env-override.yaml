service:
  environment: "production"
  replicas: 3.0
database:
  pool_size: 20.0
  timeout: "30s"
logging:
  level: "warn"
  output: "stdout"
features:
  cache: true
  analytics: true
  new_feature: true
