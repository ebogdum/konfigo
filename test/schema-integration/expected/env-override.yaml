database: {}
features:
  analytics: false
  cache: true
service:
  environment: development
  name: ENV-OVERRIDE-SERVICE
  url: ${service.protocol}://${service.name}:${service.port}
